x-pack/plugins/observability_solution/infra/public/hooks/use_lens_attributes.ts

```diff
@@ -34,7 +34,11 @@ export const useLensAttributes = (params: UseLensAttributesParams) => {
     const builder = new LensConfigBuilder(dataViews, formulaAPI);
 
     return builder.build(params) as Promise<LensAttributes>;
-  }, [params.chartType, params.dataset, dataViews]);
+    // FIXME: this should be memoized upstream, but Dario
+    // cannot find a reasonable fix, so he'll just leave
+    // this in place.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [params.chartType, params.dataset, dataViews, lens]);
```

Comment:
suggestion: https://github.com/dgieselaar/kibana/pull/56

Reply:
thanks for this!